src/ProjectService.Data/Interfaces/IRoleRepository.cs

```diff
@@ -7,12 +8,26 @@ namespace LT.DigitalOffice.ProjectService.Data.Interfaces
     /// Provides methods for working with the database of ProjectService.
     /// </summary>
     public interface IRoleRepository
-    {
+    {      
+        /// <summary>
+        /// Adds new role to the database. Returns the id of the added role.
+        /// </summary>
+        /// <param name="role">Role to add.</param>
+        /// <returns>Id of the added role.</returns>
+        Guid CreateRole(DbRole role);
+
+        /// <summary>
+        /// Returns roleId from database.
+        /// </summary>
+        /// <param name="roleId"> Role to find. </param>
+        /// <returns> Role with specified id. </returns>
+        DbRole GetRole(Guid roleId);
```

Comment:
There are no tests for this method.

Reply:
It`s not my task, this method was created by Spartak

Reply:
Had seen https://github.com/LT-Students/DigitalOffice-ProjectService/pull/27